src/components/page-details/page-details.jsx

```diff
@@ -27,14 +27,19 @@ export default class PageDetails extends React.Component {
   static getDerivedStateFromProps (props, state) {
     // Clear existing content when switching pages
     if (state.page && state.page.uuid !== props.match.params.pageId) {
-      return { page: null };
+      return {
+        page: null
+      };
```

Comment:
Did this need to be reformatted?

Reply:
This was not intentional. I've reverted the change